examples/shopify/README.md

```diff
@@ -0,0 +1,201 @@
+   and copy the Site API key. You will need it in step 3.
+
+   Then head over to [Shopify](https://www.shopify.com/) and create a store. Here is [a good starting point](https://www.shopify.com/online) to get you started.
+
+2. **Clone this template**
+
+   Instead of using `create-next-app`, run this command from the terminal:
+
+   ```bash
+   npx degit makeswift/makeswift/examples/shopify  shopify
+   cd shopify
+   ```
+
+   It will download this subdirectory of the `makeswift/makeswift` repo without including git history
+
+3. **Update environment variables**
+
+   Rename `.env.local.example` to `.env.local` and update it with values from your Shopify and Makeswift accounts.
+
+   - `SHOPIFY_STORE_NAME` can be found in the Shopify dashboard
+
+       <img src="https://user-images.githubusercontent.com/20950876/184916524-667084c1-06a0-4fa3-8f4b-73aff5a88e65.png" width="600" />
+
+   - `SHOPIFY_ACCESS_TOKEN` requires you to [register an app](https://www.shopify.com/partners/blog/17056443-how-to-generate-a-shopify-api-token)
+   - and finally the `MAKESWIFT_SITE_API_KEY` comes from your Makeswift site. It is the value you copied from step 1.
+
+   ```diff
+   - SHOPIFY_STORE_NAME=
+   - SHOPIFY_ACCESS_TOKEN=
+
+   - MAKESWIFT_SITE_API_KEY=
+   + SHOPIFY_STORE_NAME=<YOUR_SHOPIFY_STORE_NAME>
+   + SHOPIFY_ACCESS_TOKEN=<YOUR_SHOPIFY_ACCESS_TOKEN>
+
+   + MAKESWIFT_SITE_API_KEY=<YOUR_MAKESWIFT_SITE_API_KEY>
+   ```
+
+4. **Start the dev server**
+
+   Run this command from the terminal:
+
+   ```bash
+   yarn dev
+   # or
+   npm run dev
+   ```
+
+   Your next.js app should be up and running at http://localhost:3000.
+
+5. **Create a home page with a list of products**
+
+   1. Create a blank page
+
+      Look for the plus button in the left toolbar and specify "Blank page."
+
+   2. Edit the page's URL to be `/`
+
+      Hover over the "Untitled page" you just created, click on the ellipsis that appears, and click "Edit URL".
+
+   3. Drop in the Product List component
+
+      Look for the ellipsis menu in the left toolbar and drop the Product List component into Makeswift.
+
+      With the Product List component selected, update the category you want to display and the number of products in the right panels labeled 'Category' and 'Count' respectively
+
+6. **Create a product page template with product specific details**
+
+   1. Create a blank page
+
+      Look for the plus button in the left toolbar and specify "Blank page."
+
+   2. Edit page's URL to be `/__product__`
+
+      Hover over the "Untitled page" you just created, click on the ellipsis that appears, and click "Edit URL".
+
+      You should set this value to the `productTemplatePathname` from `/lib/config.ts` which comes predefined in this template as `/__product__`
+
+   3. Drop in Product specific components
+
+      Look for the ellipsis menu in the left toolbar again and drop the Product price, Product name, and Add to cart button into Makeswift.
```

Comment:
Are we using sentence case or uppercase for component labels? Above the "Product List" component is mentioned in uppercase, but here the "Product price", "Product name", and "Add to cart button" are in sentence case

Reply:
Going with sentence case